roles/openshift_gcp/tasks/frequent_log_rotation.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Is this something that we should be sharing across clouds?  Should this move into a logrotate role?

Reply:
Probably yes, ok to do as a followup?  This was originally to ensure we were rotating everything, and on cloud images we can definitely be more opinionated.

Reply:
Fine as a follow up.  Just want to share as much of this useful code as possible.